README.md

```diff
@@ -74,10 +101,10 @@ void GDN_EXPORT godot_gdnative_terminate(godot_gdnative_terminate_options *o) {
 
 /** Script entry (Registering all the classes and stuff) **/
```

Comment:
Are we sure we want to use 8-space indentation here?

Reply:
I just followed the existing usage to minimize the diff. But yes I can switch it to tabs or 4-spaces if that's better.

Reply:
I went with tabs since most of it is C code, and even for GDScript the editor defaults to tabs still.